### DISCUSSION TITLE
What is the replacement for setChunkedEncodingDisabled?

### DISCUSSION THREAD

**Discussion by thiagohora** [aws/aws-sdk-java-v2#5249] [Q&A] [answered]
I need to set ChunkedEncodingDisabled for a particular environment, and I'm using the SDK for Java v2. How can I do it?

**Comment by sugmanue** [comment, marked as answer]
Hi @thiagohora, assuming this is for S3, you can set it using the [`S3Configuration#chunkedEncodingEnabled`](https://github.com/aws/aws-sdk-java-v2/blob/ae48b9371366d5cf52bd208b9357b1da166aa974/services/s3/src/main/java/software/amazon/awssdk/services/s3/S3Configuration.java#L341C17-L341C39) class that you pass to the client.
